tasks/docs/build.js

```diff
@@ -33,6 +33,8 @@ var
     log         = tasks.log
 ;
 
+let config = configConst;
```

Comment:
Same here. Remove the config declaration form the const group and add as separate let statement
```javascript
let  config      = require('../config/docs');
```

Reply:
fixed/discussed in https://github.com/fomantic/Fomantic-UI/pull/2589#discussion_r1045226791